site/install.md

```diff
@@ -1,15 +1,556 @@
 # Create and manage the operator
```

Comment:
The file name 'operator-setup.md' or 'operator-install.md' would be more specific and more clear IMO.    In addition,  maybe 'rest.md' could become 'operator-rest.md'.    @rjeberhard what do you think?

Reply:
Hmmm... I'm all for clarity, but I don't think this is necessary.  Since this is the operator project, my guess is that most people would assume that setup.md, install.md, and rest.md were for the operator.

Reply:
Hmm, I'm still not so sure.   We also have REST, setup, and install guidance that is not specific to the operator runtime.   One can call REST on the admin server, one can setup & install a domain, one can setup k8s itself, etc, etc.   But these three files 'setup.md' 'install.md' 'rest.md' are all very specific to the operator runtime -- they don't have anything directly to do with setting up a domain or setting up k8s itself.